### PR TITLE
Fix Italian language in license chooser form

### PR DESCRIPTION
**Pull request by cristianolongo**
## Developer Certificate of Origin
```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

### CODE REVIEW

File: complete.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ foreach ($pre_reqs as $pre_req) {
 }
 
 /* Send out the translations, too. */
-echo file_get_contents('cc-translations.js.' . $gettextlang);
+echo file_get_contents('translations/cc-translations.js.' . $gettextlang);
```
Comment (hugosolar) [inline complete.js]: Why are you changing these paths? is there an error with them?

File: translations/cc-translations.js.it [hunk 2]
```diff
@@ -21,7 +21,7 @@ function cc_js_t(s) {
...
  if (s == "Slovenia") { return "Slovenia"; } 
  if (s == "Guatemala") { return "Guatemala"; } 
- if (s == "The licensor permits others to copy, distribute and transmit the work. In return, licensees may not use the work for commercial purposes — unless they get the licensor's permission.") { return "Il <em>licenziante</em> acconsente che altri copino, distribuiscano, esibiscano ed eseguano l'opera. In cambio i <em>licenziatari</em> non possono usare l'<em>opera</em> per scopi commerciali &mdash; a meno che non abbiano ottenuto il permesso del <em>licenziante</em>."; } 
+ if (s == "The licensor permits others to copy, distribute and transmit the work. In return, licensees may not use the work for commercial purposes — unless they get the licensor's permission.") { return "Il <em>licenziante</em> acconsente che altri copinoo, distribuiscano, esibiscano ed eseguano l'opera. In cambio i <em>licenziatari</em> non possono usare l'<em>opera</em> per scopi commerciali &mdash; a meno che non abbiano ottenuto il permesso del <em>licenziante</em>."; }
```
Comment (hugosolar) [inline translations/cc-translations.js.it]: Is `copinoo` a typo?

File: complete.js (hunk 1 above)
Comment (cristianolongo) [reply complete.js]: yes you can see the error during the php execution of complete.js that the file cannot be found in the web server log

File: translations/cc-translations.js.it (hunk 2 above)
Comment (cristianolongo) [reply translations/cc-translations.js.it]: yes sorry I changed for testing purposes without rolling back changes, please tell me if I shoud do a new pull request with this fix

Comment (hugosolar) [reply translations/cc-translations.js.it]: I'm so sorry for my delay to respond but you don't need to make a new pull request... you can add your commit in the same pull request.
Once you make the commit I'll accept the PR immediately.
Thank you so much for your contribution!

**Pull request merged**